src/remote.sh

```diff
@@ -1,5 +1,6 @@
 # NOTE: It is recommended that src/kw_config_loader.sh be included before this
 # file
+. "$KW_LIB_DIR/kw_config_loader.sh" --source-only
```

Comment:
Notice the comment above this line. In a few words, it is a bad idea to add this include in this file. Maybe put it in the `kw` file, before the remote include:
```
ssh|s)
      (
        . "$KW_LIB_DIR/kw_config_loader.sh" --source-only
        . "$KW_LIB_DIR/remote.sh" --source-only

        vm_ssh $@
      )
```

Reply:
Done!